pkgs/applications/editors/neovim/utils.nix

```diff
@@ -226,23 +226,6 @@ let
       ''
         mkdir -p "$out/parser"
         ln -s "${grammar}/parser" "$out/parser/${name}.so"
-
-        mkdir -p "$out/queries/${name}"
-        if [ -d "${grammar}/queries/${name}" ]; then
-          echo "moving queries from neovim queries dir"
-          for file in "${grammar}/queries/${name}"*; do
-            ln -s "$file" "$out/queries/${name}/$(basename "$file")"
-          done
-        else
```

Comment:
Actually, I need feedback. What if we kept the part below here, but the part above here we skip that because it already finds those?

Reply:
More testing would be required. I would really like to know exactly how its being included, theres a lot of files you have to jump between to get the whole picture